indicators/Smma/Smma.Models.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Skender.Stock.Indicators
+{
+    [Serializable]
+    public class SmmaResult : ResultBase
+    {
+        public decimal? Smma { get; set; }  // simple moving average
```

Comment:
update label verbiage or remove entirely, no need for labeling here, IMO

Reply:
👍